tests/kernel/tickless/tickless_concept: disco_l475_iot1 build issue

**Describe the bug**
Test link issue:
```
/local/mcu/zephyrproject/sdk/zephyr-sdk-0.11.3-beta/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: zephyr/libzephyr.a(cortex_m_systick.c.obj): in function `__NVIC_SetPriority':
/local/mcu/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include/core_cm4.h:1822: multiple definition of `z_clock_driver_init'; zephyr/libzephyr.a(stm32_lptim_timer.c.obj):/local/mcu/zephyrproject/zephyr/drivers/timer/stm32_lptim_timer.c:77: first defined here
/local/mcu/zephyrproject/sdk/zephyr-sdk-0.11.3-beta/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: zephyr/libzephyr.a(cortex_m_systick.c.obj): in function `z_clock_set_timeout':
/local/mcu/zephyrproject/zephyr/drivers/timer/cortex_m_systick.c:168: multiple definition of `z_clock_set_timeout'; zephyr/libzephyr.a(stm32_lptim_timer.c.obj):/local/mcu/zephyrproject/zephyr/drivers/timer/stm32_lptim_timer.c:187: first defined here
/local/mcu/zephyrproject/sdk/zephyr-sdk-0.11.3-beta/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: zephyr/libzephyr.a(cortex_m_systick.c.obj): in function `z_clock_elapsed':
/local/mcu/zephyrproject/zephyr/drivers/timer/cortex_m_systick.c:229: multiple definition of `z_clock_elapsed'; zephyr/libzephyr.a(stm32_lptim_timer.c.obj):/local/mcu/zephyrproject/zephyr/drivers/timer/stm32_lptim_timer.c:267: first defined here
/local/mcu/zephyrproject/sdk/zephyr-sdk-0.11.3-beta/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: zephyr/libzephyr.a(cortex_m_systick.c.obj): in function `z_timer_cycle_get_32':
/local/mcu/zephyrproject/zephyr/drivers/timer/cortex_m_systick.c:242: multiple definition of `z_timer_cycle_get_32'; zephyr/libzephyr.a(stm32_lptim_timer.c.obj):/local/mcu/zephyrproject/zephyr/drivers/timer/stm32_lptim_timer.c:295: first defined here

```

**To Reproduce**
sanitycheck --build -p disco_l475_iot1 -T tests/kernel/tickless/tickless_concept/

**Expected behavior**
No build error

--- a/dts/arm/st/l4/stm32l4.dtsi
+++ b/dts/arm/st/l4/stm32l4.dtsi
@@ -378,3 +378,7 @@
 &nvic {
 	arm,num-irq-priority-bits = <4>;
 };
+
+&systick {
+	status = "disabled";
+};

--- a/dts/arm/st/wb/stm32wb.dtsi
+++ b/dts/arm/st/wb/stm32wb.dtsi
@@ -364,3 +364,7 @@
 &nvic {
 	arm,num-irq-priority-bits = <4>;
 };
+
+&systick {
+	status = "disabled";
+};

